dds_registration/settings.py

```diff
@@ -35,21 +35,32 @@
 environ.Env.read_env(os.path.join(BASE_DIR, ".env"))
 environ.Env.read_env(os.path.join(BASE_DIR, ".env.local"))
 
-DEV = env("DEV")
-DEBUG = env("DEBUG")
-LOCAL = env("LOCAL")
+
+def random_string(length: int = 32) -> str:
+    possibles = string.ascii_letters + string.digits
+    return "".join(random.sample(possibles, length))
+
+
+DEV = env("DEV") or env("DEBUG") or env("LOCAL")
+LOCAL = DEBUG = DEV
```

Comment:
These parameters was supposed to be independent.

My reply in #66:

> DEBUG is for django internal debugging features like showing full tracebacks in the browser. LOCAL is to enable (or notify about it) the local dev server (to use 'http' url prefix instead of 'https' when we can't get it from the request object, or to enable assets preprocessing using django filters). DEV is for our (my, probably) own debugging options. Sometimes they could be used separately.

But, in general, this is not so important: it may be necessary only in the case of complex investigations for bugs on the client and server sides.